### DISCUSSION TITLE
Trying to stitch a two color design

### DISCUSSION THREAD

**Discussion by skinnybottom** [inkstitch/inkstitch#1136] [Q&A]
Sorry, I think I posted this in the wrong place yesterday.
I am trying to ink stitch a two color design. When I do the simulation the colors show up fine. But when I come to stitch it there are no thread change commands in the g code. I am exporting it as a txt file for a grbl machine
Any help greatly appreciated
Aaron

**Comment by lexelby** [comment]
@kaalleen Did we fix a problem with the gcode writer recently?

**Reply by kaalleen** [reply]
It was supposed to be fixed in the latest official release, but I missed to update pyembroidery. I suppose it should be working in the pre-release versions, except for the update-font-svg branch (this one contains an older pyembroidery version).

@skinnybottom try e.g. this one here: https://github.com/inkstitch/inkstitch/releases/tag/dev-build-kaalleen-edit-menu
Expand the assets and choose the file for your operating system and language. Color sshouldl be written as M00 (if not customized).

**Comment by lexelby** [comment]
Try one of the most recent pre-release versions.  If that doesn't fix it, let us know here and we'll file a bug and try to get it fixed.

**Comment by skinnybottom** [comment]
Thanks for looking into it. Where do I find a pre- release version? Sent from my Huawei phone-------- Original message --------From: Lex Neva ***@***.***>Date: Thu, 8 Apr 2021, 13:50To: inkstitch/inkstitch ***@***.***>Cc: skinnybottom ***@***.***>, Author ***@***.***>Subject: Re: [inkstitch/inkstitch] Trying to stitch a two color design (#1136)
Try one of the most recent pre-release versions.  If that doesn't fix it, let us know here and we'll file a bug and try to get it fixed.
